source/mainnet/net/installation/verification-instructions.rst

```diff
@@ -0,0 +1,111 @@
+.. include:: ../../variables.rst
+.. _verification-instructions:
+
+=========================
+Verification instructions
+=========================
+
+.. contents::
+   :local:
+   :backlinks: none
+   :depth: 2
+
+To find instructions on how to verify the integrity of the file you have downloaded, navigate to the respective section, which will outline the steps needed for that specific file.
```

Comment:
To find instructions on how to verify the integrity of the file you have downloaded, navigate to the respective section that outlines the steps needed for that specific file.